project/launchers.sc

```diff
@@ -36,7 +36,96 @@ def platformSuffix: String = {
   s"$arch-$os"
 }
 
-trait Launchers extends CsModule with NativeImage {
+trait Launchers extends CsModule {
+
+  trait CliNativeImage extends NativeImage {
```

Comment:
Isn't it a lot more involved than just using a specific Ubuntu version for publishing?

Reply:
Kind of, but this approach also allows to build static and "mostly static" launchers (to be run from Alpine and "distroless" Docker images respectively), that were missing here.

GitHub also deprecates its OS images from time to time, so that makes us immune to that for `ubuntu-20.04`.